e2e-testing.md

```diff
@@ -6,10 +6,14 @@ The Detox test framework is leveraged heavily to end-to-end test the Twilio Voic
 ## Setup
 
 ### Incoming Calls
-The Twilio Node.js helper library is used to make calls to the Reference App. In order for the tests to work, the process running the `detox test` command needs three environment variables, as defined in the below table.
+The Twilio Node.js helper library is used to make calls to the Reference App. In order for the tests to work, the process running the `detox test` command needs three environment variables, as defined in the below table. CI runs should use a randomly generated and unique identity for `CLIENT_IDENTITY`. For example, having the following step before running either the `detox test` or the server will suffice:
+
+```shell
+export CLIENT_IDENTITY=$(uuidgen)
```

Comment:
Should we just put the whole `android.debug` example shell?
```
ACCOUNT_SID=xxxxx \
AUTH_TOKEN=xxxx \
CLIENT_IDENTITY=$(uuidgen) \
yarn detox:test --configuration android.emu.debug
```

Reply:
Yeah wouldn't hurt. Let's do it.